Various changes (#212)
tachiyomiorg/website: src/help/faq/README.md

#### How do I enable automatic updates?
From version **0.9.0** and onwards, Tachiyomi has automatic updates for the app enabled by default. The **Preview** version also has automatic updates enabled by default.

> However, if you think you are on an outdated version, go to <Navigation item="more"/> → <Navigation item="info"/> → and click on **Check for updates**.

Reviewer: ```suggestion
However, if you think you are on an outdated version, go to <Navigation item="more"/> → <Navigation item="about"/> → and click on **Check for updates**.
```
Change this to `about` or change the key in Navigation.vue to `info`

Author: Done

#### Automatic update is not working

> Cetain devices such as MIUI have very aggressive battery saving, which can kill the app in the background. To whitelist Tachiyomi from battery saver, go to <Navigation item="more"/> → <Navigation item="settings"/> → <Navigation item="advanced"/> and click on **Disable battery optimization**.

Reviewer 2: ```suggestion
Cetain devices such as MIUI have very aggressive battery saving, which can kill the app in the background. To whitelist Tachiyomi from battery saver, go to <Navigation item="more"/> → <Navigation item="settings"/> → <Navigation item="settings_advanced"/> and click on **Disable battery optimization**.
```

Author: Done

### General

#### What are some recommended extensions?

> Due to Covid-19 quarantine, most manga sites are under heavy load. Thus we do not recommend any source at the moment. Also, Tachiyomi is not affiliated with or responsbible for any source that is down or slow.
> 
> ::: disclaimer
> Tachiyomi is not affiliated with or responsbible for any source that is down or slow.
> :::

Reviewer: ```suggestion
Due to Covid-19 quarantine, most manga sites are under heavy load. Thus we do not recommend any source at the moment. 

::: warning DISCLAIMER
Tachiyomi is not affiliated with or responsible for any source that is down or slow.
:::
```
"disclaimer" is not a build-in custom container in Vuepress, but you can use warning and danger with a title.

While Tachiyomi does not officially endorse or recommend any manga website, below listed are some popular sources reported by users. **MangaDex**, **MangaSee** and **MangaNelo** are among the most popular in general, these are large sites which host content from scanlators. Description of some popular sources are listed below:

- Fastest manga updates: **Mangadex** and scanlator sources
- All around decent: **MangaNelo** and **Mangakakalot** (both in **Mangabox** extension)
- Largest manga library: **MangaPark**
- High quality scans and official translation: **MangaSee**
- Official Shonen Jump latest chapters: **MANGA Plus by SHUEISHA**
- Webtoons and Manhwa/Manhua: **Webtoons**, **Toonily** (found in **Madara** extension), **Mangazuki.me** (in **Madara** extension), **IsekaiScan.com** (also in **Madara** extension)
- Western comics: **ReadComicOnline** and **ComicExtra**
- Yaoi, Yuri, Shounen Ai and Shoujo Ai: **Mangahasu**, **Dynasty** (for Yuri), **Lily Manga** (for Yuri, in **Madara** extension) **MyReadingManga** (for Yaoi)
- Other popular sources: **MangaLife**, **MangaOwl**, **MangaHere**, **Manhwa18.net** (in **FMReader** extension)

> Most of the scanlator sources are part of **Multi-source** extensions, which contain multiple sources. For example, extensions like **Genkan**, **Madara**, **FoolSlide** and **ComiCake** which contain sources which host directly from the scanlators' own websites.

Reviewer: I would say keep this and have a warning about it

Author: You mean this part below right?

> Most of the scanlator sources are part of **Multi-source** extensions, which contain multiple sources. For example, extensions like **Genkan**, **Madara**, **FoolSlide** and **ComiCake** which contain sources which host directly from the scanlators' own websites.

Reviewer: No, I mean keep all of it. And suggesting just to put a warning above "While Tachiyomi does..." addressing why sources can be slow now during COVID-19. By keeping it one doesn't need to rewrite it when COVID-19 ends

##### HTTP error 403
The source you selected may have Cloudflare protection on and is enforcing CAPTCHAs.

> 1. Find a manga in your library that is from the source you're trying to access.
> 2. Tap on the <Navigation item="webview"/> in the **INFO** tab.
> 3. Complete the CAPTCHA if present and wait for the page to finish loading.
> 4. Press back rather than exiting the app and if that does not work, use the back arrow.

Reviewer 2: ```suggestion
1. Find a manga in your library that is from the source you're trying to access.
1. Tap on the <Navigation item="webview"/> in the **INFO** tab.
1. Complete the CAPTCHA if present and wait for the page to finish loading.
1. Press back rather than exiting the app and if that does not work, use the back arrow.
```

Author: Done
